### ISSUE TITLE
Parser missing in Cisco Umbrella Solution

### ISSUE THREAD

**Issue by SecureHats** [Azure/Azure-Sentinel#2335]
When deploying the Cisco Umbrella solution the parser is not deployed.
The parser is also not part of the ARM template being called:
[direct link to Cisco ARM template](https://catalogartifact.azureedge.net/publicartifacts/azuresentinel.azure-sentinel-solution-ciscoumbrella-3bfe734b-8603-4172-b699-d3ab53220ba2-azure-sentinel-solution-ciscoumbrella/Artifacts/mainTemplate.json)

**To Reproduce**
1. Go to 'Solutions (Preview)'
2. Click on 'Cisco Umbrella'
3. Go to 'Logs'
4. Select Functions
5. Open 'Workspace functions'

**Expected behavior**
Expected to CiscoUmbrella to be available as this is called from the Analytics Rules

**Screenshots**
![image](https://user-images.githubusercontent.com/72928684/119267620-b731bc80-bbef-11eb-9947-9da43bd4f8c4.png)
![image](https://user-images.githubusercontent.com/72928684/119267642-d2043100-bbef-11eb-8b4d-bf5b07511466.png)

**Additional context**
Althought the parser is available in the solutions folder in the Azure Sentinel Github repository, it is not part of the ARM template. Unable to open a PR as the template is not part of the repo.

Current workaround is to manually deploy the parser

@v-ampami

**Comment by v-jayakal** [comment]
Cisco Umbrella data connector and parser are currently not part of the Cisco Umbrella solution and are considered for inclusion into solution as part of the roadmap. This issue has been noted as internal bug to track the issue. As this is planned for future, closing this issue.

**Comment by SecureHats** [comment]
Hi,
Thanks for the reply, but this basically means that this solution is not usable because all analytics, workbook and hunting queries are not working because of the missing parser

**Comment by v-jayakal** [comment]
@SecureHats , For workbook, hunting queries and analytics to work, you need data of Cisco Umbrella, For data, data connector should have already deployed (or setup) and receiving data. As part of the data connector, creation of Parser should have been done. Can you tell me is that not the case at your end? Don't you have the connector and parser set up? 
We plan move the connector and parser into solution as per the roadmap and its in the planning stage.  To unblock,  please  create a parser, link for parser with instruction steps : https://aka.ms/sentinel-ciscoumbrella-function if not done as part of the connector deployment.
Thanks.

**Comment by SecureHats** [comment]
Hi,
I have Cisco Umbrella data in my Log Analytics workspace provisioned in the custom tables (image 1)

![image](https://user-images.githubusercontent.com/72928684/119778086-58a66000-bec7-11eb-8c56-7002df885db6.png)

My expectation is that when deploying the Solution through Azure Sentinel, all components described will be deployed (image 2)

![image](https://user-images.githubusercontent.com/72928684/119778341-af139e80-bec7-11eb-8da4-91f8e7d6ee40.png)

This parser, as part of the solution, is required to return the data in the sample queries (image 3)
![image](https://user-images.githubusercontent.com/72928684/119778527-ef731c80-bec7-11eb-868e-6bed91e75c9b.png)

Because this parser is not part of the [arm template](https://catalogartifact.azureedge.net/publicartifacts/azuresentinel.azure-sentinel-solution-ciscoumbrella-3bfe734b-8603-4172-b699-d3ab53220ba2-azure-sentinel-solution-ciscoumbrella/Artifacts/mainTemplate.json) deployed which is called by the solution deployment, the solution is not working as expected.

The parses is part of the solution package, as shown in image 2, and also resides in the Cisco Umbrella solutions folder on GitHub [Cisco Umbrella Solution](https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/CiscoUmbrella) 

As seen with other solutions like Palo Alto, creating a parser manually causes the Data Connectors section in Azure Sentinel to break in case a parser already exists with the same name in another category within Log Analytics. So in what Category does this parser needs to be created? 

I am willing to create a PR to include the parser if you let me know where to create the PR for the updated ARM template. :-)

**Comment by v-jayakal** [comment]
Engineering team has a work item to check avoiding of duplicate parser installation (to avoid issues with existing parser installations). Currently, for this item, as there is no action from engineering team, removing myself from assignee list. Thanks.
